.circleci/config.yml

```diff
@@ -45,11 +49,13 @@ jobs:
       - checkout
       - restore_cache:
           keys:
-            - composer-v1-{{ checksum "composer.lock" }}
+            - composer-v1-{{ .Revision }}
             - composer-v1-
-      - run: 
+      - run:
           name: Composer install
           command: composer install -n --prefer-dist
+          environment:
```

Comment:
You could place all the environment variables on job level instead of command level, same as `CC_TEST_REPORTER_ID`. This makes env variables available for every command.

Reply:
I moved these just to keep the environment variables in one place. However, the COMPOSER_MEMORY_LIMIT only applies to `composer` which is only run in that one command in each job, so it won't have any impact on the other commands in the jobs.